paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -120,8 +122,8 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         args.config?.shippingDetails?.toConfirmPaymentIntentShipping()
     )
 
-    private val _paymentSheetResult = MutableLiveData<PaymentSheetResult>()
-    internal val paymentSheetResult: LiveData<PaymentSheetResult> = _paymentSheetResult
+    private val _paymentSheetResult = MutableStateFlow<PaymentSheetResult?>(null)
```

Comment:
Looks like this is a great candidate for `MutableSharedFlow< PaymentSheetResult>(replay = 1)` instead of making this nullable.